doc: Add MQTTBox integration guide (#3164)
TheThingsNetwork/lorawan-stack: doc/content/integrations/mqtt-clients/mqttbox/_index.md

---
title: "MQTTBox"
description: ""
weight: 
---

> [MQTTBox](http://workswithweb.com/mqttbox.html) is a cross platform application that helps with creating MQTT clients, virtual device networks, load testing MQTT devices and brokers. 

Reviewer: ```suggestion
[MQTTBox](http://workswithweb.com/mqttbox.html) is a cross platform application that makes it simple to create MQTT clients, virtual device networks, and load test MQTT devices and brokers. 
```
